changelog/6089.breaking.rst

```diff
@@ -0,0 +1 @@
+Changed the default interpolation order for :meth:`sunpy.map.GenericMap.rotate` from 4 (typically biquartic) to 3 (typically bicubic).
```

Comment:
Can this get another sentence explaining the motivation for this breaking change?